models/TotalTVL.js

```diff
@@ -0,0 +1,82 @@
+module.exports = class TotalTvl {
+    constructor (liquidityKUSD, liquidityKAR) {
+        /**
+            Generate DEX Data
+             * @param dexTvl {
+                * @param kusdKsmPair {
+                    * @param kusd
+                    * @param ksm
+                * }
+                * @param karKsmPair {
+                    * @param kar
+                    * @param ksm
+                * }
+                * // TODO : adjust values to USD
+                * @param totals {
+                    * @param dexTotal
+                    * @param karKsmTotal
+                    * @param kusdKsmTotal
+                * }
+            * }
+         */
+        this.dexTvl = {}
+        this.dexTvl.kusdKsmPair = {}
+        this.dexTvl.karKsmPair = {}
+        this.dexTvl.kusdKsmPair.kusd = this.getLiquidityValue(liquidityKUSD[0])
+        this.dexTvl.kusdKsmPair.ksm = this.getLiquidityValue(liquidityKUSD[1])
+        this.dexTvl.karKsmPair.kar = this.getLiquidityValue(liquidityKAR[0])
+        this.dexTvl.karKsmPair.ksm = this.getLiquidityValue(liquidityKAR[1])
+        this.dexTvl.totals = this.calculateDexTotals()
+
+        /**
+            Generate stablecoin TVL
+         * @param 
+        */
+
+        /**
+            Generate crowdloan TVL
+            * @param somebody
+        */
+    }
+
+    /**
+     * Takes a currency and returns a unit
+     * i.e. MKAR = 1 million KKAR = 1000
+        * @param currency
+    */
+    calculateUnit(currency) {
+        switch(currency) {
+            case 'mkar':
+                return 1000000;
+            case 'kkar':
+                return 1000;
+            default:
+                console.error("cannot find currency")
+                throw new Error('Invalid currency given to BlockTvl.calculateUnit')
+        }
+    }
+
+    /**
+     * Takes a response from dex.liquiditypool query and calculates the raw unit of currency
+     * i.e. 500 MKAR = 500 KAR * 1,000,000 = 500,000,000 KAR
+        * @param '500 MKAR'
+    */
+    getLiquidityValue(str) {
+        const currency = str.substring(str.length - 4).toLowerCase();
```

Comment:
this is unnecessary if you use `toString()` instead of `toHuman()`
`toHuman()` should only be used for display purpose only

Reply:
Looks like you are right. The 18 digit format is given when .toString is used instead of .toHuman